### ISSUE TITLE
Controller methods with models as signature not being parsed

### ISSUE THREAD

**Issue by sNok3** [dedoc/scramble#453]
As the title suggests, when generating the docs for a controller which has model binding, the schema for the model will result in just the fields that the model has, without the annotations it has.

**Comment by sNok3** [comment]
<img width="610" alt="image" src="https://github.com/user-attachments/assets/6392aace-c8d6-4ea6-82c7-bce24a47316e">

```php
class Driver extends Model
{
    use HasFactory;
    use HasStatuses;

    /**
     * The attributes that are mass assignable.
     *
     * @var array<int, string>
     */
    protected $fillable = [
        /**
         * The internal ID of the associated route
         *
         * @var int
         *
         * @example 1
         */
        'route_id',
        /**
         * The identifier of the activity
         *
         * @example 3d4b6f0e-2b0b-4b0b-8b4b-0b2b4b0b2b4b
         */
        'identifier'
   ];
```
So as you can see, I have annotations for both fields here, but under the schemas, there are is no information about the model.
<img width="320" alt="image" src="https://github.com/user-attachments/assets/53faef1b-cf74-4b17-836a-36206826bf1b">

**Comment by sNok3** [comment]
Yeah, that is exactly what I needed. However, using API resources in the controller method isn't really my way to go for now since I have a big codebase.

However, I'll gladly wait for this to get implemented!

**Comment by alecpl** [comment]
At least type and description could be fetched from the `@property` definitions on the model class.
